tests/support/activemq/subscribers_details.py

```diff
@@ -3,13 +3,18 @@
 
 import requests
 
+from django.conf import settings
 from parsel import Selector
+
 from tests.support.dtos import SubscriberSetup
 
 
-def offline_durable_subscribers(host) -> Generator[SubscriberSetup, None, None]:
+def offline_durable_subscribers(
+    host: str = settings.STOMP_SERVER_HOST,
+    port: str = settings.STOMP_SERVER_INTERFACE_PORT,
```

Comment:
**suggestion:** Same thing here. Since these params have a default value, the type could be `Optional`.